.env.example

```diff
@@ -3,3 +3,7 @@ MONGO_DEV_URI=<development-database>
 MONGO_PROD_DB=<production-database>
 MONGO_TEST_DB=<test-database>
 SECRET=<the_secret_to_sign_token>
+MAIL_USERNAME=<email-username>
+MAIL_PASSWORD=<email-password>
```

Comment:
Please remember to remove unused environment variables,

Since you are using the `OAuth`, password is not needed